nservicebus/upgrades/5to6.md

```diff
@@ -442,12 +442,14 @@ snippet: 5to6CriticalError
 
 ## Notifications
 
-The `BusNotifications` class previously exposed the available notification hooks as observables implementing `IObservable`. This meant a custom `IObserver` and the use of [Reactive-Extensions](https://msdn.microsoft.com/en-au/data/gg577609.aspx) was required to use this API. 
+The `BusNotifications` class has been renamed to `Notifications`. 
 
-In Versions 6 and above the notifications API has been changed `BusNotifications` class has been renamed to `Notifications`. It now also exposes regular events instead of observables for easier usage. Find out more about [subscribing to error notifications](/nservicebus/errors/subscribing-to-error-notifications.md). To continue using Reactive-Extensions the events API can be transformed into `IObservable`s like this:
+`BusNotifications` previously exposed the available notification hooks as observables implementing `IObservable`. This meant a custom `IObserver` and the use of [Reactive-Extensions](https://msdn.microsoft.com/en-au/data/gg577609.aspx) was required to use this API. With Version 6 the notifications API has been changed to expose regular events instead of observables for easier usage. To continue using Reactive-Extensions the events API can be transformed into `IObservable`s like this:
 
 snippet: ConvertEventToObservable
 
+Notification subscriptions can now also be registered at configuration time on the `EndpointConfiguration.Notifications` property. See the [error notifications documentation](http://docs.particular.net/nservicebus/errors/subscribing-to-error-notifications) for more details and samples.
```

Comment:
links should be either relative (starting with `./` or `../`) or absolute (stating with `/`)